### DISCUSSION TITLE
How to use credentials in kafka-versions.yaml

### DISCUSSION THREAD

**Discussion by chandans21j** [strimzi/strimzi-kafka-operator#8165] [Q&A]
How to use credentials in kafka-versions.yaml

**Comment by vishalsingh9990** [comment]
No we cannot use it

**Reply by scholzj** [reply]
I'm not sure what credentials you want to use or why. But there is no such support today. The Apache servers do not require any credentials.

**Reply by chandans21j** [reply]
Actually, i have put kafka in some other url and that url i have linked into kafka-version.yaml,But that  url is secure with login credentials.
so how to provide login credentials so that kafka can be used.
Hope this will clear you.

**Reply by scholzj** [reply]
Well, as I said - there is no support for anything like that because we do not need anything like that. If you use something simple such HTTP Basic Authentication, you can try to encode it in the URL ... e.g. `https://user:pwd@my.tld/kafka-3.4.0.zip`. But as I said, I don't think nobody ever tried it.
